package.json

```diff
@@ -56,7 +56,9 @@
 		"docker:env": "node -e \"require('fs').createWriteStream( 'docker/.env', { flags: 'a' } );\"",
 		"docker:compose-extras": "[ -f docker/compose-extras.yml ] || cp docker/compose-extras.yml.sample docker/compose-extras.yml",
 		"docker:update-core-unit-tests": "yarn docker:compose exec wordpress svn up /tmp/wordpress-develop/tests/phpunit/data/ /tmp/wordpress-develop/tests/phpunit/includes",
-		"lint": "eslint --ext .js,.jsx . _inc extensions modules",
+		"lint": "npm-run-all --parallel lint:*",
+		"lint:js": "eslint --ext .js,.jsx . _inc extensions modules",
+		"lint:scss": "stylelint \"**/*.scss\" --syntax scss",
 		"lint-file": "eslint --ext .js,.jsx",
```

Comment:
`lint-file` looks a bit odd next to others now 🤷‍♂️  https://github.com/Automattic/jetpack/pull/15723